app/models/presenter/bid.rb

```diff
@@ -81,6 +82,10 @@ def bidder_name
       def amount
         NULL
       end
+
+      def raw_amount
+        nil
```

Comment:
Well, I thought it would be advantageous to return a Presenter::Bid object in several places, but I don't think `Presenter::Bid::Null.nil?` returns true and it probably shouldn't?